packages/sdks/snippets/angular/src/app/blog-article/blog-article.component.ts

```diff
@@ -0,0 +1,41 @@
+import { CommonModule } from '@angular/common';
+import { Component } from '@angular/core';
+import { Content, fetchOneEntry } from '@builder.io/sdk-angular';
+
+@Component({
+  selector: 'app-blog-article',
+  standalone: true,
+  imports: [Content, CommonModule],
+  template: `<div *ngIf="article" class="content">
+    <h1>{{ article.data.title }}</h1>
+    <p>{{ article.data.blurb }}</p>
+    <img [src]="article.data.image" alt="" />
+    <content [model]="model" [content]="article"></content>
+  </div> `,
+})
+export class BlogArticleComponent {
+  article: any;
+  model = 'blog-article';
+
+  constructor() {
+    this.article = {
+      data: {
+        title: 'Sample Blog Title',
+        blurb: 'This is a sample blurb for the blog article.',
+        image: 'path/to/sample-image.jpg',
+      },
+    };
+  }
```

Comment:
why is this placeholder needed? can't we instantiate `this.article` only in `ngOnInit`?

Reply:
You are right; there is no point in having this constructor and instantiating this.article there. It will work even if we don't initialize them because they come directly from the visual editor. I have updated the code and pushed it. Kindly check.